### PR TITLE
Improve docs for aws_ssm_association schedule_expression

### LINKED ISSUE #27141: [Docs]: aws_ssm_association needs an example for schedule_expression
### Documentation Link

https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/ssm_association

### Description

The input parameter `schedule_expression` could do with some practical examples to make it easier to use.
Current documentation doesn't provide example values or an indication of what values are accepted.

### References

_No response_

### Would you like to implement a fix?

Yes

### PR DESCRIPTION
**Pull request by baolsen**
### Description

Improve docs for `aws_ssm_association` resource `schedule_expression` parameter.

### Relations

Closes #27141

### CODE REVIEW

**Comment by bschaatsbergen** [comment]
Thanks for raising this pull request @baolsen, how about we turn the last sentence into a practical example within the resource documentation?

I'm referring to:

> Example values: `"cron(0 2 ? * SUN *)"`, `"cron(30 23 * * ? *)"` or `"rate(7 days)"`. See [AWS reference documentation](https://docs.aws.amazon.com/systems-manager/latest/userguide/reference-cron-and-rate-expressions.html).

I would suggest to remove this and add an additional example block like: https://github.com/hashicorp/terraform-provider-aws/blob/d9a58c52d2f209a0cf1abe1c12b220c8b3278042/website/docs/r/ssm_association.html.markdown?plain=1#L43-L56.

**Review by bschaatsbergen — COMMENTED** [review]
Left a comment regarding a practical example.

**Comment by baolsen** [comment]
Updated and ready for review.

File: website/docs/r/ssm_maintenance_window.html.markdown [hunk 1]
```diff
@@ -26,7 +26,7 @@ resource "aws_ssm_maintenance_window" "production" {
 The following arguments are supported:
```
Comment (baolsen) [inline website/docs/r/ssm_maintenance_window.html.markdown:27]: Dead link fixed

**Review by jar-b — APPROVED** [review]
LGTM 👍

**Comment by jar-b** [comment]
Thanks for your contribution @baolsen!

**Pull request merged**